dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/programrule/engine/ProgramRuleExpressionEvaluator.java

```diff
@@ -36,9 +39,26 @@
  */
 public class ProgramRuleExpressionEvaluator implements RuleExpressionEvaluator
 {
+    private static final Log log = LogFactory.getLog( ProgramRuleExpressionEvaluator.class );
+
     @Override
     public String evaluate( String expression )
     {
-       return ExpressionUtils.evaluate( expression ).toString();
+        String result = "";
+
+        try
+        {
+            result = ExpressionUtils.evaluate( expression ).toString();
+        }
+        catch ( JexlException je )
+        {
+            result = "false";
```

Comment:
Would it better to throw and catch some exception here, instead of swallowing it? Or return null? Or a constant?